### DISCUSSION TITLE
SD and 2 passes - best practices?

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1749] [Q&A]
Hi,

as we have a two pass pipeline now for rendering stuff, any ideas on how to get the best results with these? Like which samling method and how many sampling steps is a good pick? Start with a lower number of steps on first pass and then more on second? Keep it the same?

I have no understanding of what's going on in the background.

**Comment by vladmandic** [comment]
Discord channel has far more suggestions.
We'll update the wiki with some findings.

**Reply by Tillerz** [reply]
You linked a discord once, I added it, it's named Automatic1111's Server. Is that the one? Cause I cannot see any discussion on there about it. :)

**Reply by Tillerz** [reply]
Found it.
